Dockerfile.parity

```diff
@@ -19,4 +19,6 @@ WORKDIR /
 FROM keydonix/parity-instantseal
 COPY --from=builder /parity/chains /parity/chains
 
+COPY --from=builder /maker-deployer/output/contracts /contracts
```

Comment:
Consider also adding the generated TypeScript files for Maker?  Would be nice if they were more generic (not dependent on `eth.js`, but we can address that in the future.